docs/overview/architecture.md

```diff
@@ -15,17 +16,61 @@ Vald uses a cloud-native architecture focusing on [Kubernetes](https://kubernete
 Some components in Vald use Kubernetes API to control the behavior of distributed vector indexes.
 Before reading this document, you need to have some understanding of the basic idea of cloud-native architecture and Kubernetes.
 
-The below image is Vald's architecture.
+### Concept
 
-<img src="../../design/Vald Future Architecture Overview.svg" />
+Here are the concepts of Vald.
+
+- Microservice based
+
+  Vald is designed based on the microservice architecture. Vald components are highly decoupled into small components and connected, which increases the overall agility and maintainability of Vald.
+
+- Containerized
+
+  All components in Vald are containerized, which means you can easily deploy Vald components on any environment.
+
+- Observability & Real-time monitoring
+
+  Vald supports pprof and Prometheus to monitor and trace the log in every Vald components.
+
+- Distributed
+
+  All the data and index are distributed to Vald Agents in the Vald cluster so whenever you search a vector, all Vald agents can process parallelly and merge the result by Vald LB Gateway.
```

Comment:
![badge](https://img.shields.io/badge/review-imo-orange.svg?style=flat-square)
agents don't know "all the data". they know just UUIDs and indices.

I think "distributed" is not so important, but "distributed vector spaces" is an important aspect of the architecture.

Reply:
Yes you are correct. What do you think the following changes?
```suggestion
  All the vector data and indexes are distributed to Vald Agents in the Vald cluster. Whenever you search a vector in Vald cluster, all Vald agents can process parallelly and merge the result by Vald LB Gateway.
```